R/local-timer.R

```diff
@@ -0,0 +1,28 @@
+#' Local Timer
+#'
+#' @inheritParams params
+#'
+#' @return xx
+#' @export
+#'
+#' @examples
+#' local_timer()
+local_timer <- function(.local_envir = rlang::caller_env()) {
+  chk_env(.local_envir)
+  
+  if(!requireNamespace("withr", quietly = TRUE)) {
+    err("Package 'withr' must be installed to create a local_timer().")
+  }
+  
+  tmr <- tmr_(seconds  = 0, start = TRUE, caller = sys.call(-1))
```

Comment:
```suggestion
  tmr <- tmr_start(as_hms(0))
```

and remove `tmr_()` for now? Or simplify accordingly?